task/06-conditions-n-loops-tasks.js

```diff
@@ -82,7 +99,12 @@ function getSumBetweenNumbers(n1, n2) {
  *   10,10,10 =>  true
  */
 function isTriangle(a,b,c) {
-    throw new Error('Not implemented');
+    if(a < b + c && b < a + c && c < a + b){
+        return true;
+    }
+    else{
+        return false;
+    }
```

Comment:
We can simply return (a < b + c && b < a + c && c < a + b)
Please, see https://javascript.info/logical-operators